wallet/build.gradle

```diff
@@ -41,63 +42,95 @@ dependencies {
+    implementation 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
 
     implementation project(path: ':common')
-    annotationProcessor 'androidx.room:room-compiler:2.2.5'
+//    implementation project(path: ':ui-redesign')
+    implementation 'com.google.android.gms:play-services-auth:18.1.0'
+
+    def room_version = "2.2.5"
+    implementation "androidx.room:room-ktx:$room_version"
+    implementation "androidx.room:room-runtime:$room_version"
+    annotationProcessor "androidx.room:room-compiler:$room_version"
 
     //Uphold Integration
     implementation project(path: ':uphold-integration')
 
     implementation 'androidx.multidex:multidex:2.0.1'
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.3.2'
+    implementation "androidx.core:core-ktx:1.3.2"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8"
 
     implementation 'com.github.bumptech.glide:glide:4.11.0'
     annotationProcessor 'com.github.bumptech.glide:compiler:4.11.0'
 
-    implementation 'com.google.firebase:firebase-analytics:17.5.0'
+    implementation 'com.github.MikeOrtiz:TouchImageView:3.0.3'
+
+    implementation 'com.google.firebase:firebase-analytics:17.6.0'
+    implementation 'com.google.firebase:firebase-inappmessaging-display:19.1.1'
+
+    // Import the BoM for the Firebase platform
+    implementation platform('com.google.firebase:firebase-bom:26.1.1')
+
+    // Declare the dependencies for the Crashlytics and Analytics libraries
+    // When using the BoM, you don't specify versions in Firebase library dependencies
+    implementation 'com.google.firebase:firebase-crashlytics'
+    implementation 'com.google.firebase:firebase-analytics'
+
+    // Declare the dependency for the Performance Monitoring library
+    // When using the BoM, you don't specify versions in Firebase library dependencies
+    //implementation 'com.google.firebase:firebase-perf-ktx'
 
-    implementation ('com.google.firebase:firebase-inappmessaging:19.1.1') {
-        exclude group: 'com.google.protobuf', module: 'protobuf-lite'
-        exclude group: 'com.google.firebase', module: 'protolite-well-known-types'
+    // Google Drive Dependencies
+    implementation('com.google.api-client:google-api-client-android:1.26.0') {
+        exclude group: 'com.google.guava', module: 'guava-jdk5'
     }
-    implementation ('com.google.firebase:firebase-inappmessaging-display:19.1.1') {
-        exclude group: 'com.google.protobuf', module: 'protobuf-lite'
-        exclude group: 'com.google.firebase', module: 'protolite-well-known-types'
+    implementation('com.google.apis:google-api-services-drive:v3-rev173-1.25.0') {
+        exclude group: 'com.google.guava', module: 'guava-jdk5'
+    }
+
+    implementation('com.google.http-client:google-http-client-gson:1.26.0') { // do we need this
+        exclude group: 'com.google.guava', module: 'guava-jdk5'
```

Comment:
This section looks like it was copied from our `dashpay` or `evonet` branches.  Can we return this to the original version or are some of these changes required to support liquid?